logux-vue-todomvc/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "logux-vue-todomvc",
+  "version": "0.1.0",
```

Comment:
Do we use `version` here? It is not mandatory with `private`. But if we want to use `version` we need some policy what is public API.